config/presets/SMART_RunClientAgainstServer.json

```diff
@@ -0,0 +1,31 @@
+{
+  "title": "Demo: Run Against the SMART Server Suite",
+  "id": "smart_run_client_against_server_v2_2",
+  "test_suite_id": "smart_client_stu2_2",
+  "inputs": [
+    {
+      "name": "smart_jwk_set",
+      "description": "The SMART client's JSON Web Key Set. May be provided as either a publicly accessible url containing the JWKS, or the raw JWKS.",
+      "optional": true,
+      "title": "SMART JSON Web Key Set (JWKS)",
+      "type": "textarea",
+      "value": "http://localhost:4567/custom/smart_stu2_2/.well-known/jwks.json"
```

Comment:
Rather than hardcoding these to localhost, you make these `.json.erb` files and use `Inferno::Application['base_url']` here to allow these presets to work both locally and in deployed environments.

Reply:
fixed